emmo/ontology.py

```diff
@@ -69,7 +69,7 @@ def get_ontology(self, base_iri='emmo-inferred'):
             of EMMO
         """
         if base_iri == 'emmo':
-            base_iri = 'http://emmo.info/emmo'
+            base_iri = 'https://emmo-repo.github.io/latest-stable/emmo.owl'
```

Comment:
By loading EMMO as a single file we will loose the possibility to easily list the individual ontologies separately as well as the documentation of the individual ontologies. On the positive side, it will load much faster. It is up to us to decide. Probably this is a good choice.

Reply:
I think importing separate files is the most correct for this. Otherwise the inferred version can be uploaded. I changed the uri to the raw as http://emmo.info/emmo did not work